.github/workflows/website.yml

```diff
@@ -1,6 +1,7 @@
 name: Website
 
 on:
+  workflow_dispatch:
```

Comment:
Are you sure there should be a colon at the end?

Reply:
Let me double check 👀

Reply:
An alternative way to do it is to use a tilde `~` that apparently is valid for `null` in tag resolution according to the YAML spec.

https://yaml.org/spec/1.2.2/#1032-tag-resolution

```suggestion
  workflow_dispatch: ~
```

Reply:
I take it back. It does work as it is 💯 
_I tested in another repo_ ✅ 

The following lines 👇 

https://github.com/techeverri/techeverri-cli/blob/5daf89d0b3f8ccdc39339bec7398b0f41619987c/.github/workflows/ci.yml#L6-L7

Make the workflow to have a manual trigger in the user interface

https://github.com/techeverri/techeverri-cli/actions/workflows/ci.yml
<img width="963" alt="image" src="https://user-images.githubusercontent.com/3720424/160950667-12bc678f-284d-41af-975c-9b543dca2d06.png">

Reply:
> Are you sure there should be a colon at the end?

Yes. Final answer 👍

Reply:
This syntax is also used in the official docs from GitHub for when the trigger doesn't have any config.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-data-from-the-triggering-workflow

<img width="785" alt="image" src="https://user-images.githubusercontent.com/3720424/160950923-add96a66-27f1-4b13-be6e-5bdc548f10cf.png">